deployment/constructor_params/lynx/lynx-alpha-13-child-params.json

```diff
@@ -0,0 +1,27 @@
+{
+  "LynxMockTACoApplication": {},
+  "ProxyAdmin": {},
+  "LynxTACoChildApplication": {
+    "_rootApplication": "$LynxMockTACoApplication"
+  },
+  "TransparentUpgradeableProxy": {
+    "_logic": "$LynxTACoChildApplication",
+    "admin_": "$ProxyAdmin",
+    "_data": "$EMPTY_BYTES"
```

Comment:
Can you use just `""`?

Reply:
Unfortunately not, since the type must be a bytes literal.

Reply:
I mean, in the same way that `"$EMPTY_BYTES"` is not a bytes literal but you process it to `b""`.

Reply:
Passing bytes in general as a parameter is an area that we need to implement - the only thing we do for now are empty bytes, we don't allow any actual bytes to be provided. 

An area that needs more work in the long-term.